packages/util/src/environment.ts

```diff
@@ -138,6 +138,7 @@ export function isNodeSdk(): boolean {
 export function isSafari(): boolean {
   return (
     !isNode() &&
+    navigator.userAgent !== undefined &&
```

Comment:
My apologies... could you replace `navigator.userAgent !== undefined` with simply `navigator.userAgent`? That style is more conventional in our code base. Thanks!